tests/unit_tests/test_mesh_to_vtk.py

```diff
@@ -77,8 +77,8 @@ def test_write_data_to_vtk_size_mismatch(mesh):
     # by regex. These are needed to make the test string match the error message
     # string when using the match argument as that uses regular expression
     expected_error_msg = (
-        f"The size of the dataset 'label' \({len(data)}\) should be equal to "
-        f"the number of mesh cells \({mesh.num_mesh_cells}\)"
+        f"The size of the dataset 'label' ({len(data)}) should be equal to "
+        f"the number of mesh cells ({mesh.num_mesh_cells})"
```

Comment:
I noticed the tests were failing and I recall that the "\" was needed to help the regex compare the expected_error_msg when using regex

pytest.raises makes use of regex and I believe this is why the tests are not passing.


```suggestion
        f"The size of the dataset 'label' \({len(data)}\) should be equal to "
        f"the number of mesh cells \({mesh.num_mesh_cells}\)"
```

Reply:
Thanks for pointing that out (should have read the comment above :man_facepalming:). I've added back the slashes but also made it a raw string to avoid the warnings about escaping characters that don't make sense.